.github/workflows/check-terraform-module-version.yaml

```diff
@@ -8,11 +8,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check for editted files for terraform modules
-        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@add-gh-action-to-pin-terraform-modules-to-latest 
+      - name: Check edited files for terraform modules
+        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@main
         env:
           REPO_NAME: cloud-platform-environments
-          API_URL: ${{ env.TERRAFORM_MODULE_VERSIONS_API_URL }}
+          API_URL: ${{ vars.TERRAFORM_MODULE_VERSIONS_API_URL }}
```

Comment:
Previously, we couldn't access the env var as we were using the wrong context, to access configuration variables we need to use the vars context.